28_FFTBloom/main.cpp

```diff
@@ -875,7 +880,7 @@ class FFTBloomApp final : public examples::SimpleWindowedApplication, public app
 		uint16_t firstAxisFFTWorkgroupSizeLog2;
 		smart_refctd_ptr<IGPUShader> shaders[3];
 		{
-			auto [elementsPerInvocationLog2, workgroupSizeLog2] = workgroup::fft::optimalFFTParameters(m_device->getPhysicalDevice()->getLimits().maxOptimallyResidentWorkgroupInvocations, m_marginSrcDim.height);
+			auto [elementsPerInvocationLog2, workgroupSizeLog2] = workgroup::fft::optimalFFTParameters(m_device->getPhysicalDevice()->getLimits().maxOptimallyResidentWorkgroupInvocations, m_marginSrcDim.height, m_device->getPhysicalDevice()->getLimits().minSubgroupSize);
```

Comment:
https://github.com/Devsh-Graphics-Programming/Nabla-Examples-and-Tests/pull/171/files#r1923739241